sys/include/ztimer/xtimer_compat.h

```diff
@@ -131,6 +137,23 @@ static inline void xtimer_nanosleep(uint32_t nanoseconds)
     ztimer_sleep(ZTIMER_USEC, nanoseconds / NS_PER_US);
 }
 
+static inline void xtimer_tsleep32(xtimer_ticks32_t ticks)
+{
+    xtimer_usleep(xtimer_usec_from_ticks(ticks));
+}
+
+static inline uint64_t xtimer_tsleep64(xtimer_ticks64_t ticks)
+{
+    const uint32_t max_sleep = UINT32_MAX;
+    uint64_t time = xtimer_usec_from_ticks64(ticks);
+
+    while (time > max_sleep) {
+        xtimer_usleep(clock, max_sleep);
```

Comment:
there is a bug in this line 
`clock` should be `ZTIMER_USEC`

Reply:
If you open a PR fixing this I noted that what could also be added:

- dummy XTIMER_BACKOFF
- xtimer_init
- xtimer_is_set
- xtimer_rmutex_lock_timeout

Reply:
right i will add them